vcpkg/users/versioning-troubleshooting.md

```diff
@@ -188,6 +188,57 @@ Resolutions:
       run: git fetch --unshallow
     ```
 
+## <a name="unexpected-default-features"></a> Cause: Unexpected installation of default features in transitive dependencies
+
+When using `vcpkg` to manage your project's dependencies, you might encounter a situation where a transitive dependency installs with its default features, even when you've specified `"depend-defaults": false` in your top-level manifest. This can lead to confusion when features that were intended to be excluded, such as `foo`, are still being installed.
+
+### Scenario
+
+You have a dependency on library `Y`, which in turn depends on library `X`. Library `X` has default features, including `foo`, that you want to exclude from your project. Your top-level manifest for library `Y` might look something like this:
+
+```json
+{
+  "name": "my-project",
+  "dependencies": [
+    {
+      "name": "Y",
+      "features": ["featureB"],
+      "depend-defaults": false
+    }
+  ]
+}
+```
+
+You expect that `X` will be installed without its default features due to the `"depend-defaults": false` setting. However, `X` is still installed with the default feature `foo`.
+
+### Reason
+
+The `depend-defaults` property is only considered at the top-level manifest. This means that default features of transitive dependencies (like `X` in this scenario) are still included unless explicitly disabled at the top level. When library `Y` is resolved, `vcpkg` does not propagate the `"depend-defaults": false` setting to the transitive dependency `X`, resulting in the installation of `X` with its default features.
```

Comment:
> ... only considered at the top level manifest ...

I observe that port authors consider the port's manifest as being the top-level manifest when running commands like `vcpkg install my-port`. And this interpretation is fairly reaonable IMO.  
However, it is not compatible with the proposed wording.  Thus it is necessary to avoid `top-level manifest`.

(I'm really pessimistic about the outlook to permanently solve this confusion with documentation alone. It is really scatching the principle of least surprise if the behavior for one top-level manifest is different from the behavior for another top-level manifest.)